asaim.yaml

```diff
@@ -0,0 +1,338 @@
+---
+install_repository_dependencies: true
+install_resolver_dependencies: true
+install_tool_dependencies: false
+
+tools:
+
+# data_retrieval
+- name: 'suite_ebi_tools'
+  owner: 'iuc'
+  tool_panel_section_label: 'Get Data'
+  
+- name: 'suite_enasearch'
+  owner: 'iuc'
+  tool_panel_section_label: 'Get Data'
+
+- name: 'suite_ncbi_entrez'
+  owner: 'iuc'
+  tool_panel_section_label: 'Get Data'
+
+- name: 'sra_tools'
+  owner: 'iuc'
+  tool_panel_section_label: 'Get Data'
+
+# text_manipulation 
+- name: 'add_value'
+  owner: 'devteam'
+  tool_panel_section_label: 'Text Manipulation'
+
+- name: 'change_case'
+  owner: 'devteam'
+  tool_panel_section_label: 'Text Manipulation'
+
+- name: 'column_join'
+  owner: 'jjohnson'
+  tool_panel_section_label: 'Text Manipulation'
+
+- name: 'collection_column_join'
+  owner: 'iuc'
+  tool_panel_section_label: 'Text Manipulation'
+
+- name: 'column_maker'
+  owner: 'devteam'
+  tool_panel_section_label: 'Text Manipulation'
+
+- name: 'concatenate_multiple_datasets'
```

Comment:
Is this really needed? I think this can be replaced by either the text-processing tools or the new data-collection tools.

Reply:
Which data-collection tools?